scss/table.scss

```diff
@@ -148,4 +148,10 @@ tr.bx--expandable-row-v2 + tr[data-child-row] td {
 
 .bx--row {
   width: 100%
+}
+
+.addResourceLink {
```

Comment:
This css class name is not how I normally see them named.  I think avoid using verbs like add and call it a resourceLink or emptyTableResourceLink if that describes it better.

Reply:
That makes sense. I'll change it to 'emptyTableResourceLink'.